### ISSUE TITLE
order of optimizer.step() and lr_scheduler.step() in tests

### ISSUE THREAD

**Issue by pmeier** [pmeier/pystiche_papers#114]
Unit tests raise the following warning

```
tests/ulyanov_et_al_2016/test_utils.py::test_DelayedExponentialLR
  /home/runner/work/pystiche_papers/pystiche_papers/.tox/tests/lib/python3.6/site-packages/torch/optim/lr_scheduler.py:123: UserWarning: Detected call of `lr_scheduler.step()` before `optimizer.step()`. In PyTorch 1.1.0 and later, you should call them in the opposite order: `optimizer.step()` before `lr_scheduler.step()`.  Failure to do this will result in PyTorch skipping the first value of the learning rate schedule. See more details at https://pytorch.org/docs/stable/optim.html#how-to-adjust-learning-rate
    "https://pytorch.org/docs/stable/optim.html#how-to-adjust-learning-rate", UserWarning)
```

@jbueltemeier Please look into it.

**Comment by jbueltemeier** [comment]
The warning only occurred in the test, because there it was only tested if the` lr_scheduler` was working correctly, for which only the `lr_scheduler.step()` is needed. So it is not important here whether the `optimizer.step()` is called or not.

**Comment by pmeier** [comment]
Fixed in #120.
